third-party/rspamd/src/test/java/org/apache/james/rspamd/client/RspamdHttpClientTest.java

```diff
@@ -221,6 +236,46 @@ void checkNonVirusMailUsingRspamdClientWithExactPasswordShouldReturnHasNoVirus()
         assertThat(analysisResult.hasVirus()).isFalse();
     }
 
+    @Disabled
+    @Test
+    void concurrentTest() {
```

Comment:
Should we move this to `DockerRspamdExtensionTest` and enable it as a reminder that Rspamd learning throughput is limited? And IMO we should change the default FeedHam/FeedSpamd' `messagesPerSecond`.

Reply:
> And IMO we should change the default FeedHam/FeedSpamd' messagesPerSecond.

Which number we should set by default?
This depend on a resource of server

Reply:
Ok, this could be the issue of resource management rather Rspamd itself.